### DISCUSSION TITLE
[dep-check] react-native-linear-gradient banned notice

### DISCUSSION THREAD

**Discussion by PaitoAnderson** [microsoft/rnx-kit#1685] [General] [answered]
Is there a particular issue you could link to regarding this? I see it's unmaintained, but I was unaware of the startup performance issue and I'm curious which platform.

https://github.com/microsoft/rnx-kit/blob/aa0a4fcd2699bb5cefed7b96a4eee3ef0a2e3a44/packages/dep-check/src/presets/banned.ts#L20

**Comment by tido64** [comment, marked as answer]
Hi @PaitoAnderson, this is something we've seen in some of our apps. It's significant enough that we've banned it. If I recall correctly, it does a bunch of work on startup, even before the bridge is up and running. It basically regressed our TTI. I'm not sure whether we investigated the root cause though.

**Reply by PaitoAnderson** [reply]
Thanks @tido64. I appreciate the quick reply.

**Comment by rborn** [comment]
@tido64 may I ask what do you use as alternative? 🤗

**Reply by tido64** [reply]
@crazyfraggle: Do you recall what you ended up replacing `react-native-linear-gradient` with?

**Reply by crazyfraggle** [reply]
We simply removed gradient from the design, and stopped using the package if I recall correctly.
An option that I would consider as alternative would be react-native-svg. Though I'm not sure about gradient performance there as I haven't tried it out.

**Reply by rborn** [reply]
@tido64 @crazyfraggle Thanks a lot for the replies 🥰 I was thinking on svg too, we use it in the app ( and don't think we'll get rid of it soon) so we'll probably migrate to the svg gradient 🙏

**Reply by tido64** [reply]
FWIW, we've just released [1.13.0](https://github.com/microsoft/rnx-kit/releases/tag/%40rnx-kit/dep-check%401.13.0) with a revised message.
